config/locales/en.yml

```diff
@@ -1106,6 +1104,15 @@ en:
     recently_reviewed: Recently Reviewed
 
   activerecord:
+    models:
```

Comment:
We should probably move these into `en.models.yml`. If we put these under `errors`, we could possibly use `self.errors.generate_message(:price_policy, :missing)`. That might be problematic for the null objects, though, and it might be overkill.